internal/runners/clean/config.go

```diff
@@ -52,14 +53,19 @@ func (c *Config) Run(params *ConfigParams) error {
 		return locale.NewError("err_clean_cache_activated")
 	}
 
-	if !params.Force {
-		ok, err := c.confirm.Confirm(locale.T("confirm"), locale.T("clean_config_confirm"), ptr.To(true))
-		if err != nil {
-			return locale.WrapError(err, "err_clean_config_confirm", "Could not confirm clean config choice")
-		}
-		if !ok {
-			return locale.NewInputError("err_clean_config_aborted", "Cleaning of config aborted by user")
-		}
+	defaultChoice := !c.confirm.IsInteractive()
```

Comment:
nit: Could probably get rid of this with `ptr.To(!c.confirm.IsInteractive()` below

Reply:
We could, but the in-line argument wouldn't be as descriptive. This reads better and falls in line with other `Confirm()` calls.